src/openrpc/certificationextension.json

```diff
@@ -0,0 +1,383 @@
+                        "required": [
+                            "state"
+                        ],
+                        "properties": {
+                            "method": {
+                                "type": "string",
+                                "description": "The method name"
+                            },
+                            "state": {
+                                "type": "boolean",
+                                "description": "The state of the response whether it is success or failure"
+                            }
+                        },
+                        "description": "JSON object that represents the state of the interaction response"
+                    }
+                },
+                "examples": [
+                    {
+                        "interactionRequest": {
+                            "id": "1",
+                            "jsonrpc": "2.0",
+                            "method": "token",
+                            "params": {
+                                "type": "platform"
+                            }
+                        },
+                        "interactionResponse": {
+                            "id": "1",
+                            "jsonrpc": "2.0",
+                            "state": true
+                        }
+                    }
+                ]
+            },
+            "PlatformIntent": {
+                "title": "PlatformIntent",
+                "description": "A Firebolt compliant representation of a user intention to interact to a specific place in an app or control the media",
+                "anyOf": [
+                    {
+                        "$ref": "https://meta.comcast.com/firebolt/intents#/definitions/DeviceIntent"
+                    }
+                ]
+            },
+            "HealthCheckResult": {
+                "title": "HealthCheckResult",
+                "description": "Performance and healthcheck snapshot for the device and process which includes load, memory, cpu usage. It also provides total live firebolt connections.",
+                "type": "object",
+                "required": [
+                    "load",
+                    "processMemory",
+                    "cpu",
+                    "deviceMemory",
+                    "connections"
+                ],
+                "properties": {
+                    "load": {
+                        "type": "number",
+                        "description": "The load average of the system"
+                    },
+                    "processMemory": {
+                        "type": "number",
+                        "description": "The memory usage of the process"
+                    },
+                    "cpu": {
+                        "type": "number",
+                        "description": "The cpu usage of the system"
+                    },
+                    "deviceMemory": {
```

Comment:
What do you think about using other adjectives on these values?  It's reasonable that we might want to expand this object to return more fields later on, so it might be helpful to be clear in our responses now e.g.

```
cpuLoad1minAverage
cpuLoad5minAverage
deviceMemoryUsed
memoryUsed
```